docs/products/postgresql/howto/pgbouncer-stats-datadog.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Expose PgBouncer statistics to Datadog
+sidebar_label: PgBouncer stats in Datadog
+---
+
+Connect PgBouncer with Datadog to display [PgBouncer statistics](/docs/products/postgresql/howto/pgbouncer-stats) on the Datadog platform.
```

Comment:
I recommend "Integrate/Connect PgBouncer with Datadog to track connection pool metrics, and monitor application traffic on the Datadog platform".